examples/cmd/run.py

```diff
@@ -12,6 +12,13 @@
 import wave
 import ChatTTS
 from IPython.display import Audio
+import numpy as np
+
+try:
+    from fire import Fire
```

Comment:
It'd better not to import new package. Just use the built-in `parser`
```py
import argparse

parser = argparse.ArgumentParser(description='ChatTTS cmd demo')
parser.add_argument(
    "--stream", action="store_true", help="Use stream infer"
)
args = parser.parse_args()
```
Then use it with
```py
args.stream # True or False
```